force-app/main/default/classes/MonthlyIterator_TEST.cls

```diff
@@ -0,0 +1,416 @@
+@IsTest
```

Comment:
I love that we have so many tests but I'm wondering if we should use a more varied start date, or add additional tests with different start dates.

Reply:
sure